test/extended/util/framework.go

```diff
@@ -1400,3 +1400,20 @@ func CreateExecPodOnNode(client kcoreclient.CoreInterface, ns, nodeName, name st
 	o.Expect(err).NotTo(o.HaveOccurred())
 	return created.Name
 }
+
+func CheckForEvent(client kcoreclient.CoreInterface, build *buildapi.Build, reason, message string) {
```

Comment:
Give this a name that indicates it's checking for build events, or move it to a build specific package.